contracts/FeesAndBootstrapRewards.sol

```diff
@@ -95,7 +97,8 @@ contract FeesAndBootstrapRewards is IFeesAndBootstrapRewards, ManagedContract {
         updateGuardianFeesAndBootstrap(guardian);
         uint256 amount = feesAndBootstrap[guardian].bootstrapBalance;
         feesAndBootstrap[guardian].bootstrapBalance = 0;
-        emit BootstrapRewardsWithdrawn(guardian, amount);
+        feesAndBootstrap[guardian].withdrawnBootstrap = feesAndBootstrap[guardian].withdrawnBootstrap.add(amount);
```

Comment:
can assign the updated withdrawnBootstrap  in a memory variable

Reply:
Done